tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+	"extends": "./node_modules/@balena/lint/config/tslint-prettier.json",
+	"rules": {}
+}
```

Comment:
Atm this is only used as a way to point IDEs to the correct lint rules.
Project specific rules can be added here but these should only be limited to things like prohibiting whole `lodash`/`rendition` imports so that consumer projects can produce smaller combined build outputs.